www/docs/alert/index.php

```diff
@@ -341,6 +341,38 @@ function display_search_form ( $alert, $details = array(), $errors = array() ) {
             echo '<em class="error">You have used a comma in your search term &ndash; are you sure this is what you want?
 You cannot sign up to multiple search terms using a comma &ndash; either use OR, or fill in this form multiple times.</em>';
         }
+        if ( preg_match('#([A-Z]{1,2}\d+ ?\d[A-Z]{2})#i', $alertsearch, $m) && strlen($alertsearch) > strlen($m[1]) && validate_postcode($m[1]) ) {
+            $scottish_text = '';
+            $mp_display_text = '';
+            if (postcode_is_scottish($m[1])) {
+                $mp_display_text = 'your MP, ';
+                $scottish_text = ' or MSP';
+            }
```

Comment:
Though of course you have multiple MLAs for a postcode, don't you. I guess it could sign up for any of them saying the thing?